haas/api.py

```diff
@@ -611,6 +630,8 @@ def show_network(network):
 
     if network.access is not None:
         result['access'] = network.access.label
+    else:
+        result['access'] = 'Public'
 
     return json.dumps(result)
```

Comment:
Please add `sort_keys=True`. Not sure if it's needed, but since we're in the code anyway, might as well be consistent.